sdk/spring/azure-spring-data-gremlin/README.md

```diff
@@ -0,0 +1,193 @@
+  - `@EdgeSet` maps to a set of `Edge`
+  - `@EdgeFrom` maps to the head `Vertex` of an `Edge`
+  - `@EdgeTo` maps to the tail `Vertex` of an `Edge`
+  - `@Graph` maps to an `Object` to a `Graph`
+- Supports advanced operations 
+  - `<T> T findVertexById(Object id, Class<T> domainClass);`
+  - `<T> T findEdgeById(Object id, Class<T> domainClass);`
+  - `<T> boolean isEmptyGraph(T object)`
+  - `long vertexCount()`
+  - `long edgeCount()`
+- Supports [Spring Data custom query](https://docs.spring.io/spring-data/commons/docs/current/reference/html/#repositories.query-methods.details) find operation, e.g.,  `findByAFieldAndBField`
+- Supports any class type in domain class including collection and nested type.
+
+
+### Spring Data Version Support
+This repository only supports Spring Data 2.x. Version mapping between spring boot and spring-data-gremlin: 
+
+| Spring boot version                                         | spring-data-gremlin version                                                                                                                                                                                                                   |
+|:-----------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| ![version](https://img.shields.io/badge/version-2.3.x-blue) | [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.spring.data.gremlin/spring-data-gremlin/2.3.svg)](https://search.maven.org/search?q=g:com.microsoft.spring.data.gremlin%20AND%20a:spring-data-gremlin%20AND%20v:2.3.*) |
+| ![version](https://img.shields.io/badge/version-2.2.x-blue) | [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.spring.data.gremlin/spring-data-gremlin/2.2.svg)](https://search.maven.org/search?q=g:com.microsoft.spring.data.gremlin%20AND%20a:spring-data-gremlin%20AND%20v:2.2.*) |
+| ![version](https://img.shields.io/badge/version-2.1.x-blue) | [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.spring.data.gremlin/spring-data-gremlin/2.1.svg)](https://search.maven.org/search?q=g:com.microsoft.spring.data.gremlin%20AND%20a:spring-data-gremlin%20AND%20v:2.1.*) |
+| ![version](https://img.shields.io/badge/version-2.0.x-blue) | [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.spring.data.gremlin/spring-data-gremlin/2.0.svg)](https://search.maven.org/search?q=g:com.microsoft.spring.data.gremlin%20AND%20a:spring-data-gremlin%20AND%20v:2.0.*) |
+
+## Getting started
+
+### Add the dependency
+`spring-data-gremlin` is published on Maven Central Repository.  
+If you are using Maven, add the following dependency.  
+
+[//]: # ({x-version-update-start;com.microsoft.spring.data.gremlin:spring-data-gremlin;current})
```

Comment:
This groupId/artifactId is not correct, this is the external dependency.

Reply:
Thanks for pointing it out, will update it.